example/android/app/build.gradle

```diff
@@ -71,15 +71,11 @@ android {
             signingConfig signingConfigs.release
 
             minifyEnabled true
-            useProguard true
 
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
         debug {
-            signingConfig signingConfigs.release
-
             minifyEnabled true

```

Comment:
As a note, for `debug` there's no need to set m`inifyEnabled` to true and add proguard.